.cascade-code/Chapman.edu/_components/omni_nav_v2/omni_nav.vtl

```diff
@@ -17,35 +17,33 @@
 ## umbrellaCategories
 ## These are locations for which OmniNav should build the secondary off-canvas-nav menu.
 ## umbrellaCategory is the term product owners use.
-## Does order matter here? I'm assuming it does.
-## TODO: Ross is reviewing the labels and will provide an updated list.
 ## I would prefer that each inner array item within the array were only 2 items and
 ## all other vars be derived from those:
 ## [$umbrellaFolderId, $umbrellaLabel]
 ## Current inner array values:
-## [ $umbrellaDirectory, $umbrellaLabel/$umbrellaBrandLabel, $umbrellaClass, $umbrellaHref ]
+## [ $umbrellaDirectory, $umbrellaLabel/$umbrellaBrandLabel, $umbrellaClass ]
```

Comment:
the only thing the last param was being used for was for the links but product owners want all umbrella categories to link to their main index page

Reply:
👍